examples/nextjs-blog/utils/post/post.util.js

```diff
@@ -14,3 +15,10 @@ export function getDateString(timeStampWithZone) {
 
   return format(datetime, 'DD MMMM YYYY')
 }
+
+export function getDateFromTimestamp(timestamp) {
+  // TODO: Confirm what the best way is to get a date from a timestamp object (not instance)
+  return firestoreService.Timestamp.fromMillis(
```

Comment:
added a fix for this - seems timestamp comes back as anobject and a Timestamp object

one is `{ _seconds: xxx, ... }` and the other is `Timestamp { seconds: xxx, .... }`

Reply:
Thanks. I'm still not sure what the recommended way is to work with these timestamps because this just feels like the "hard/wrong" way.